script/Arcueid/Day 5/05_00_ARC05_2.txt

```diff
@@ -181,7 +181,7 @@ Then that hotel tragedy...
 [sha:578ea89ba499e2419f54076694aef6c3102db2ae]{
 -- Page 187, Offset 9043.
 -- ホテルの人間を殺したのはヴローヴでも、その災害を呼びこんだのは自分だと。
-Although Vlov was the one that killed those people, we were the ones that called down that disaster.
+Although Vlov was the one that killed those people, we were the ones that brought forth that disaster.
```

Comment:
I think I prefer called down, brought forth suggests they had just a slightly more active hand in it